src/main/java/org/terasology/launcher/util/HostServicesWrapper.java

```diff
@@ -0,0 +1,70 @@
+package org.terasology.launcher.util;
+
+import javafx.application.HostServices;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+
+/**
+ * Provide capabilities to interact with the hosting system (OS).
+ * <p>
```

Comment:
what is `<p>` used for? can this be removed?

Reply:
See [How To Write Doc Comments](https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format) for more details. In short, you can use HTML tags in JavaDoc such that it renders nicely.